py/server/deephaven/experimental/iceberg.py

```diff
@@ -245,3 +246,73 @@ def adapter_aws_glue(
     except Exception as e:
         raise DHError(e, "Failed to build Iceberg Catalog Adapter") from e
 
+
+def adapter(
```

Comment:
All of the other methods are named `adapter_<something>`.  Should this have a similar format?  e.g. `adapater_properties` or `adapter_config`